packages/components/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,267 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon } from '@frontify/fondue-icons';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useState } from 'react';
+
+import { Button } from '../Button/Button';
+
+import { SelectCombobox } from './Combobox';
+import { Select, SelectInput } from './Select';
+import { SelectItem, SelectItemGroup } from './SelectItem';
+import { SelectSlot } from './SelectSlot';
+import { type SelectItemType } from './useSelectData';
+
+type Story = StoryObj<typeof meta>;
+const meta: Meta<typeof SelectInput> = {
+    component: SelectInput,
+    subcomponents: {
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Combobox': SelectCombobox,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Slot': SelectSlot,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Group': SelectItemGroup,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Item': SelectItem,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        'aria-label': 'Select an item',
+    },
+};
+export default meta;
+
+export const SimpleSelect: Story = {
+    name: 'Select',
+    render: (args) => {
+        return (
+            <Select {...args}>
+                <Select.Slot name="menu">
+                    <Select.Item value="test1">Test1</Select.Item>
+                    <Select.Item value="test2">Test2</Select.Item>
+                    <Select.Item value="test3">Test3</Select.Item>
+                </Select.Slot>
```

Comment:
We shouldn't have to put the slot `menu` if there are no slot (default)
```suggestion
                <Select.Item value="test1">Test1</Select.Item>
                <Select.Item value="test2">Test2</Select.Item>
                <Select.Item value="test3">Test3</Select.Item>
```

Reply:
added